core/extension/gdextension.h

```diff
@@ -136,6 +136,15 @@ class GDExtension : public Resource {
 		INITIALIZATION_LEVEL_EDITOR = GDEXTENSION_INITIALIZATION_EDITOR
 	};
 
+protected:
+#ifndef DISABLE_DEPRECATED
+	Error _open_library_bind_compat_88418(const String &p_path, const String &p_entry_symbol);
```

Comment:
```suggestion
	Error _open_library_bind_compat_91502(const String &p_path, const String &p_entry_symbol);
```
Should be named for the PR that added them to make it easier to track